scripts/create-package/plop-templates-react/package.json.hbs

```diff
@@ -2,7 +2,6 @@
   "name": "{{{packageNpmName}}}",
   "version": "0.1.0",
   "description": "{{{description}}}",
-  "private": true,
```

Comment:
You should bring this back--we actually still want to avoid publishing new packages by default.